control/control.leanos.xml

```diff
@@ -233,9 +233,7 @@ textdomain="control"
         <clone_module>ssh_import</clone_module>
         <clone_module>tftp-server</clone_module>
         <clone_module>security</clone_module>
-<!--
-        <clone_module>scc</clone_module>
--->
+        <clone_module>suse_register</clone_module>
```

Comment:
Well, I still think it needs a bit comment why it is OK to clone it as we either do not clone reg keys or we clone it, but I am not sure if it is ok.

Reply:
I am checking when the security team.

Reply:
As discussed in https://bugzilla.suse.com/show_bug.cgi?id=1174202#c2 we already export some sensitive data. So we have been asked to set 0o600 permissions on the generated file (see https://github.com/yast/yast-autoinstallation/pull/707).